### DISCUSSION TITLE
Executing fixed_time operation only once

### DISCUSSION THREAD

**Discussion by MathiasMantai** [frePPLe/frepple#632] [Q&A]
Hello again,
I'm trying to get something very specific to work: I have maschines that need to be prepared for manufacturing. After preparing the maschine will produce a certain amount of an item (e.g. 5000)
In frepple I have modeled this with an operation of type routing consisting of 2 operations, 1 fixed_time and 1 time_per operation. The problem is, when frepple splits the order into multiple parts while planning the fixed_time operation is executed multiple times before every time_per operation. This however isn't really practical since a maschine has to be prepared only once and can then produce all the items.

Is there any way to model this in frepple, meaning that a fixed_time operation is executed once even if the consecutive time_per operation is split up and executed more than once?

**Comment by jdetaeye** [comment]
The modeling with a routing isn't the simplest modeling.
A single time_per operation would be sufficient:   the total duration of a time_per operation operation is "duration + quantity * duration per unit", so it can already model a constant overhead.

It is correct that this constant overhead will be repeated for every manufacturing order.
There are 2 approaches to this:
- Leave it like this, and just work with some average overhead time.   For > 80 % of businesses this is an acceptable approximation for planning.
- It the changeover times are really big, optimizing the plan by reducing the number of setup changes is an important objective of the plan. See https://frepple.com/docs/current/user-guide/examples/resource/resource-setup-matrices.php on the "setup matrix" functionality.   Important to know: a) it's a pretty complex modeling, and b) only supported in the Enterprise&Cloud Editions of frepple.
